public/app/plugins/datasource/prometheus/querybuilder/shared/OperationEditor.tsx

```diff
@@ -40,6 +40,8 @@ export function OperationEditor({
   const styles = useStyles2(getStyles);
   const def = queryModeller.getOperationDef(operation.id);
 
+  console.log('rendering OperationEditor', operation.params);
```

Comment:
Assuming you don't want to keep the console.log?

Reply:
yea :) forgot to remove it